frontend/packages/kubevirt-plugin/integration-tests/tests/cnv-installation.scenario.ts

```diff
@@ -0,0 +1,25 @@
+import * as sidenavView from '@console/internal-integration-tests/views/sidenav.view';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { $, browser, by, element } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+
+const namespaceButton = $('.co-namespace-selector button');
+const openshiftNamespaceButton = $('#openshift-cnv-link');
+
+ 
+describe('Go to operators page', () => {
+ 
+  beforeAll(async () => {
+    await sidenavView.clickNavLink(['Operators', 'OperatorHub']);
+    await isLoaded();
+    await click(namespaceButton);
+    await click(openshiftNamespaceButton);
+  });
+ 
+  it('install Kubevirt', async () => {
+    await isLoaded();
+    element(by.cssContainingText('.catalog-tile-pf-title', 'Container-native virtualization Operator')).click();
```

Comment:
This would be better to define in a view file

Reply:
How?

Reply:
1) Create cnv-installation.view.ts file under `views` directory.
2) In cnv-installation.view.ts:

```
export const cnvOperator = element(by.cssContainingText('.catalog-tile-pf-title', 'Container-native virtualization Operator'));
```
3) in cnv-installation.scenario.ts
```
import { cnvOperator } from 'path to that view file';
....

await click(cnvOperator);
```